crates/ruff_cli/src/cache.rs

```diff
@@ -161,11 +180,50 @@ impl Cache {
             return None;
         }
 
+        file.last_seen
+            .store(self.last_seen_cache, Ordering::Release);
```

Comment:
Nit: Wouldn't `Relaxed` be sufficient?

Reply:
Maybe, but it can also be subtly wrong. So I'm more inclined to use the stronger orderings unless lessening it is proven correct (and faster).

Reply:
My understanding is that `Release` should be used together with `Aquire` (which we are not?) [source](https://marabos.nl/atomics/memory-ordering.html#release-and-acquire-ordering)